### DISCUSSION TITLE
Sorry for the trivial things... Widgets and chart

### DISCUSSION THREAD

**Discussion by maurizioaiello** [YetiForceCompany/YetiForceCRM#17114] [How to]
Sorry for the trivial thing, 
but on demo I can see chart and widgets and that's exactly what I search.

I went through all the menus both in user mode and in admin mode, but I cannot realize how to do that, and no howto explain it. 

Can somebody help me please?
Thanks

**Comment by rskrzypczak** [comment]
You searched hard :)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/252aeb53-57bd-4926-bed2-aa12fb818d71)

**Reply by maurizioaiello** [reply]
In gitstable.yetiforce.com appears different...
Anyway, I understand I must create a filter before, but 1) what is the logic behind? and 2) both in gitstable and in my local installation if I try to add a new filter it remains stuck on "Loading please wait..."
On gitstable I can edit, but on my local installation I am completely stopped.

What is wrong?

Thanks
